tests/fast-integration/skr-test/provision/provision-skr.js

```diff
@@ -17,13 +17,19 @@ async function provisionSKRInstance(options, timeout) {
     console.log(`\nInstanceID ${options.instanceID}`,
         `Runtime ${options.runtimeName}`, `Application ${options.appName}`, `Suffix ${options.suffix}`);
 
+    const customParams = {
+      oidc: options.oidc0,
+      kymaVersion: 'PR-14312',
```

Comment:
Why does this need to be tested with a specific PR? Are the features used here not on the main branch?

Reply:
yes, I want to test my changes related to the audit log threshold.